src/main/java/uk/gov/hmcts/reform/cosapi/exception/DocumentUploadOrDeleteException.java

```diff
@@ -1,9 +1,9 @@
 package uk.gov.hmcts.reform.cosapi.exception;
 
-public class DocuementUploadOrDeleteException extends RuntimeException {
+public class DocumentUploadOrDeleteException extends RuntimeException {
     private static final long serialVersionUID = 7442994120484411078L;
 
-    public DocuementUploadOrDeleteException(String message, Throwable cause) {
+    public DocumentUploadOrDeleteException(String message, Throwable cause) {
```

Comment:
What is the change in this file?

Reply:
It's the name of the class - wrong spelling corrected